robotframework_reportportal/service.pyi

```diff
@@ -13,7 +13,7 @@ class RobotService:
     rp: Optional[ReportPortalService] = ...
     def __init__(self) -> None: ...
     def _get_launch_attributes(self, cmd_attrs: List) -> List: ...
-    def init_service(self, endpoint: Text, project: Text, uuid: Text, log_batch_size: int, pool_size: int, skipped_issue:bool) -> None: ...
+    def init_service(self, endpoint: Text, project: Text, uuid: Text, log_batch_size: int, pool_size: int, skipped_issue: bool, verify_ssl: bool) -> None: ...
```

Comment:
It's not binary, it's a string: https://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification
You can specify certificate path to the `verify` parameters of requests library. I believe we must provide this option instead of just ability to disable SSL.

That's why I closed previous review. Because it was True/False too.